src/date-picker/date-picker-dialog.jsx

```diff
@@ -44,6 +45,7 @@ const DatePickerDialog = React.createClass({
     wordings: React.PropTypes.object,
     disableYearSelection: React.PropTypes.bool,
     initialDate: React.PropTypes.object,
+    inline: React.PropTypes.bool,
```

Comment:
I would rather not use a bool type, since we may add the popover container. Maybe we should call this property `container` with a value between `dialog` (default) and `inline`. Soon `popover`.